### PR TITLE
Fixed enable/disable command bugs 

### PR DESCRIPTION
**Pull request by hokiepokedad2**
There were errors execute the update query when executing either the enable or disable commands. The 'id' values weren't getting passed in. 

## Description
Included the 'id' value in their respective poracleMessage commands. Also fixed enumeration of mentions in the enable command. 

## Motivation and Context
Needed commands to work. 

## How Has This Been Tested?
Allow rate limiter function to stop/disable the user. Attempt to perform enable. 

## Types of changes

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/lib/poracleMessage/commands/disable.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ exports.run = async (client, msg, args, options) => {
 		for (const id of targets) {
 			client.log.info(`Disable ${id}`)
 
-			await client.query.updateQuery('humans', { admin_disable: 1, disabled_date: null }, { id })
+			await client.query.updateQuery('humans', { admin_disable: 1, disabled_date: null }, { id: id })
```
Comment (jfberry) [inline src/lib/poracleMessage/commands/disable.js:25]: Doesn't this just get changed back by lint? It certainly would by webstorm. It's equivalent to what was there before

File: src/lib/poracleMessage/commands/enable.js [hunk 2]
```diff
@@ -17,12 +17,12 @@ exports.run = async (client, msg, args, options) => {
 		// Make list of ids
 		const mentions = msg.getMentions()
 		const targets = mentions.map((x) => x.id)
-		targets.push(args.filter((x) => parseInt(x, 10)))
+		targets.push(...(args.filter((x) => parseInt(x, 10))))
```
Comment (jfberry) [inline src/lib/poracleMessage/commands/enable.js:20]: This is absolutely a bug (this change mirrors disable)

File: src/lib/poracleMessage/commands/disable.js (hunk 1 above)
Comment (hokiepokedad2) [reply src/lib/poracleMessage/commands/disable.js]: You're right. This was test code to attempt to isolate what was going on. This has been reverted.

File: src/lib/poracleMessage/commands/enable.js (hunk 2 above)
Comment (hokiepokedad2) [reply src/lib/poracleMessage/commands/enable.js:20]: Do you mean you agree that it was a bug or the fix is a bug?

Comment (jfberry) [reply src/lib/poracleMessage/commands/enable.js:20]: I agree that your change fixes the bug! Sorry if I was unclear

Comment (hokiepokedad2) [reply src/lib/poracleMessage/commands/enable.js:20]: Ah okay, no problem!

**Pull request merged**